src/test/java/com/cloudant/tests/ViewsTest.java

```diff
@@ -258,6 +472,8 @@ private static void checkPage(Page page, int docCount, int docsPerPage, int page
      * we expect. This test uses the same {@link View} instance for each page query.
      * Page forward to the last page, back to the first page, forward to the last page and back to
      * the first page.
+     *
+     * Run the same check above on an array of multi-value key views.
```

Comment:
I think removing this javadoc and adding a javadoc to each pagination multi-value key test is best.